vmdk_plugin/utils/test_util/VolumePropertiesVerificationUtil.go

```diff
@@ -0,0 +1,125 @@
+
+package e2e
+
+import (
+	"log"
+	"strings"
+)
+
+var ADMIN_CLI_LS = "/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls "
+var DOCKER_CLI_INSPC = "docker volume inspect "
+
+// returns attached to vm field of volume using docker cli
+func GetVmAttachedToVolUsingDockerCli(volName string, hostname string) string {
+	cmd := DOCKER_CLI_INSPC + " --format '{{index .Status \"attached to VM\"}}' " + volName
+	op := ExecCmd(hostname, cmd)
+	// return strings.TrimSpace(string(op))
+	if string(op) == "" {
+		log.Fatal("Null value is returned by docker cli when looking for attached to vm field for volume. Output: ", op)
+	}
+	return strings.TrimSpace(op)
+}
+
+// returns attached to vm field of volume using admin cli
+func GetVmAttachedToVolUsingAdminCli(volName string, hostname string) string {
+	cmd := ADMIN_CLI_LS + "-c volume,attached-to 2>/dev/null | grep " + volName
+	op := ExecCmd(hostname, cmd)
+	volProps := strings.Fields(op)
+	if len(volProps) != 2 {
+		log.Println("Actual admin cli output when looking for attached to vm field - ", string(op))
+		log.Fatal("Admin cli output is expected to consist of two elements only - volume name and attached-to-vm status")
+	}
+	if string(op) == "" {
+		log.Fatal("Null value is returned by admin cli when looking for attached to vm field for volume. Output: ", op)
+	}
+	return string(volProps[1])
+}
+
+// returns capacity, attached-to-vm and disk-format field
+// for volume using Admin cli
+func GetVolumePropertiesAdminCli(volName string, hostname string) string {
+	cmd := ADMIN_CLI_LS + "-c volume,attached-to,capacity,disk-format 2>/dev/null | grep " + volName
+	op := ExecCmd(hostname, cmd)
+	if len(strings.Fields(op)) != 4 {
+		log.Println("Actual admin cli output when looking for, size, disk-format and attached to vm - ", string(op))
+		log.Fatal("Output is expected to consist of four elements only - volume name, attached-to-vm status, size and disk-format")
+	}
+	if string(op) == "" {
+		log.Fatal("Null value is returned by admin cli when looking for, size, disk-format and attached to vm. Output: ", op)
+	}
+	// return string(op)
+	return op
+}
+
+// returns capacity,  attached-to-vm and disk-format field
+// for volume using Docker cli
+func GetVolumePropertiesDockerCli(volName string, hostname string) string {
+	cmd := DOCKER_CLI_INSPC + " --format '{{index .Status.capacity.size}} {{index .Status.diskformat}} {{index .Status \"attached to VM\"}}' " + volName
+	op := ExecCmd(hostname, cmd)
+	expctedLen := 0
+	// converting the output to an array and comparing the length of array is expected
+	if strings.Contains(string(op), "<no value>") {
+		expctedLen = 4
+	} else {
+		expctedLen = 3
+	}
+	if len(strings.Fields(op)) != expctedLen {
+		log.Println("Actual docker cli inspect output when looking for size, disk-format and attached to vm - ", string(op))
+		log.Fatal("Docker cli inpect output is expected to consist of three elements only - size, disk-format and attached-to-vm status")
```

Comment:
the overall message looks weird Why to Prinln and then Fatal instead of single Fatal , multi-lined ? That applies to multiple places where PrintLn();Fatal() is used